index.html

```diff
@@ -1,25 +1,34 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <!--awesome font to get the icons -->
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">    <!--custom made css -->
-    <link rel="stylesheet" href="css/style.css">    
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <!--custom made css -->
+    <link rel="stylesheet" href="css/style.css">
     <title>My Portifolio</title>
 </head>
+
 <body>
     <!--Welcome Page Start-->
     <section class="home-wrapper">
+        <div id="myMenuModal" class="menu-modal">
```

Comment:
- [ ] As required, I kindly suggest that you ensure that the mobile menu isn't visible on desktop and that When the user clicks (or taps) any option of the mobile menu, the mobile menu disappears.

<img width="763" alt="image" src="https://user-images.githubusercontent.com/52934128/194965724-4bc14cc0-27fe-464f-b003-222129bc1e86.png">